CourseKit/Source/Utils/UIUtils.swift

```diff
@@ -377,4 +377,54 @@ public class UIUtils {
             let countryDialingCode = prefix[countryRegionCode]
             return countryDialingCode!
         }
+    
+    public static func showToast(message: String) {
+        guard let window = UIApplication.shared.windows.first(where: { $0.isKeyWindow }) else { return }
+        
+        let toastLabel = createToastLabel(message: message)
+        positionToastLabel(toastLabel, in: window)
+        animateToast(toastLabel, in: window)
+    }
+    
+    private static func createToastLabel(message: String) -> UILabel {
+        let label = UILabel()
+        label.backgroundColor = UIColor.black.withAlphaComponent(0.6)
+        label.textColor = UIColor.white
+        label.textAlignment = .center
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.text = message
+        label.alpha = 0
+        label.layer.cornerRadius = 10
+        label.clipsToBounds = true
+        label.numberOfLines = 0
+        return label
+    }
+    
+    private static func positionToastLabel(_ label: UILabel, in window: UIWindow) {
+        let maxSize = CGSize(width: window.frame.size.width - 40, height: 100)
+        var expectedSize = label.sizeThatFits(maxSize)
+        expectedSize.width += 20
+        expectedSize.height += 20
+        
+        label.frame = CGRect(
+            x: window.frame.size.width/2 - expectedSize.width/2,
+            y: window.frame.size.height - expectedSize.height - 100,
+            width: expectedSize.width,
+            height: expectedSize.height
+        )
+    }
+    
+    private static func animateToast(_ label: UILabel, in window: UIWindow) {
+        window.addSubview(label)
+        
+        UIView.animate(withDuration: 0.3, delay: 0, options: .curveEaseIn, animations: {
+            label.alpha = 1
+        }, completion: { _ in
+            UIView.animate(withDuration: 0.3, delay: 1.5, options: .curveEaseOut, animations: {
+                label.alpha = 0
+            }, completion: { _ in
+                label.removeFromSuperview()
+            })
+        })
+    }
```

Comment:
Since we are not going to call these methods directly, can these be moved somewhere else.
Otherwise can me put a class for showing toast and put these methods inside that.